packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -25,14 +26,14 @@ const trace = makeTracer('StakingAccountHolder');
 const { Fail } = assert;
 /**
  * @typedef {object} StakingAccountNotification
- * @property {string} address
+ * @property {ChainAddress['address']} address
  */
 
 /**
  * @typedef {{
  *  topicKit: RecorderKit<StakingAccountNotification>;
- *  account: ChainAccount;
- *  chainAddress: string;
+ *  account: ChainAccountKit['account'];
+ *  address: ChainAddress['address'];
```

Comment:
Why are we not including the chainID as well?  If we have staking reports that are Eth, it should clear from the log?

I would love to have a standard simple format that encompasses all addresses, but we just aren't seeing it yet.

Reply:
We have #9162 for work around the `ChainAddress` object. I will stub that out now in this PR so we can remove the `ChainAccountKit['account']` references.